README.md

```diff
@@ -4,11 +4,26 @@
 
 _Run a Python API in this ready-to-use-repository in minutes_
 
-With this template respository you can quickly get hands-on a Python project for running an HTTP API using the [Fastapi](https://fastapi.tiangolo.com/) framework. It will let you focus in working with the project instead of setting and configuring anything.
+With this template respository, you can quickly start up a Python web app that serves an HTTP API using the [FastAPI](https://fastapi.tiangolo.com/) framework. 
+
+<details>
+   <summary><strong>Learn more about APIs</strong></summary>
+
+   An API (Application Programming Interface) describes a way for two computers to interact.
+   An HTTP API allows an Internet-connected computer to send an HTTP request to another Internet-connected computer
+    and receive a response. For example, my computer could send a request to
+   `http://fakeweatherwebsite.com/api/city=Los+Angeles` and receive back data like `{"high": 72, "low": 66}`.
+   
+   HTTP APIs often provide either data or functionality that's unique to a service, like the example API for the weather website. User-facing websites like Twitter and Instagram also provide APIs to retrieve back user data (provided
```

Comment:
Sometimes these websites don't require authentication to collect user data and we haven't explained authentication at this point. Could the sentence of User-facing websites be changed or dropped?